### PR TITLE
Added native support for Kudu extension #1097

### PR DESCRIPTION
**Pull request by aldettinger**
[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

File: integration-tests/kudu/src/main/java/org/apache/camel/quarkus/component/kudu/it/KuduInfrastructureTestHelper.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import java.net.InetAddress;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+
+import io.quarkus.runtime.StartupEvent;
+import org.jboss.logging.Logger;
+
+/**
+ * In order to run Kudu integration tests, {@KuduResource}, {@code KuduTest} and {@code KuduIT} should have access to:
```
Comment (oscerd) [inline integration-tests/kudu/src/main/java/org/apache/camel/quarkus/component/kudu/it/KuduInfrastructureTestHelper.java:30]: About this: we need to mention Windows is not supported.

File: .github/workflows/pr-build.yaml [hunk 2]
```diff
@@ -206,11 +206,12 @@ jobs:
               sap-netweaver
               servicenow
               slack
-          - category: SQL
+          - category: Database
```
Comment (jamesnetherton) [inline .github/workflows/pr-build.yaml:209]: If you rename these, you also need to tweak the references in the `matrix.category` array.

File: integration-tests/kudu/src/main/java/org/apache/camel/quarkus/component/kudu/it/KuduInfrastructureTestHelper.java (hunk 1 above)
Comment (aldettinger) [reply integration-tests/kudu/src/main/java/org/apache/camel/quarkus/component/kudu/it/KuduInfrastructureTestHelper.java:30]: Hey @oscerd , thanks for taking a look. Do you mean that Apache Kudu couldn't run on Windows ?

Comment (oscerd) [reply integration-tests/kudu/src/main/java/org/apache/camel/quarkus/component/kudu/it/KuduInfrastructureTestHelper.java:30]: yes, https://kudu.apache.org/docs/installation.html

File: .github/workflows/pr-build.yaml (hunk 2 above)
Comment (aldettinger) [reply .github/workflows/pr-build.yaml:209]: Ok, I found it. Thanks for the tips @jamesnetherton

File: integration-tests/kudu/src/main/java/org/apache/camel/quarkus/component/kudu/it/KuduInfrastructureTestHelper.java (hunk 1 above)
Comment (aldettinger) [reply integration-tests/kudu/src/main/java/org/apache/camel/quarkus/component/kudu/it/KuduInfrastructureTestHelper.java:30]: Okay, I'm gonna add a comment for this. Still it should be possible to use the container based mode.

**Comment by aldettinger** [comment]
Ok, thanks for review. I've pushed fixes on master.

**Pull request closed without merging**